src/pages/student-class-page/student-class-page.tsx

```diff
@@ -44,11 +44,15 @@ export function StudentClassPage({ mode, ...props }: StudentClassPageProps) {
   // const [selectedIds, setSelectedIds] = useState([0]);
   // const [selectedStudents, setSelectedStudents] = useState<StudentSelect[]>([]);
 
-  const [studentClasses, setStudentClasses] = useState(Array<StudentClass>);
-  const [classIndex, setClassIndex] = useState(0);
   const [modalState, setModalState] = useState(false);
```

Comment:
Mudaria o nome dessa variável
modalState = true/false n é mto significativo lendo (eu entendo que é ele aberto ou fechado, mas se for o caso meteria um isModalOpen, setIsModalOpen